src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/PartAdmissionResponseContentProvider.java

```diff
@@ -104,6 +108,8 @@ public PartAdmissionResponseContentProvider(
         );
 
         content.put("formNumber", ADMISSIONS_FORM_NO);
+        partAdmissionResponse.getDirectionsQuestionnaire().ifPresent(dq ->
+            content.put("hearingContent", hearingContentProvider.mapDirectionQuestionnaire.apply(dq)));
```

Comment:
mapDirectionQuestionnaire.apply in tests is fine but in code a functional method should be used as the assignment target for a lambda expression or method reference.